pretty_gpx/city/data/rivers.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+"""Rivers"""
+import os
+import numpy as np
+
+
+from matplotlib.patches import Polygon
+from shapely import Polygon as ShapelyPolygon
+
+from pretty_gpx.common.gpx.gpx_bounds import GpxBounds
+from pretty_gpx.common.gpx.gpx_data_cache_handler import GpxDataCacheHandler
+from pretty_gpx.common.data.overpass_request import overpass_query
+from pretty_gpx.common.utils.pickle_io import read_pickle
+from pretty_gpx.common.utils.pickle_io import write_pickle
+from pretty_gpx.common.utils.logger import logger
+from pretty_gpx.common.data.overpass_processing import Surface_Polygons
+from pretty_gpx.common.data.overpass_processing import create_patch_collection_from_polygons
+from pretty_gpx.common.data.overpass_processing import get_polygons_from_relations
+from pretty_gpx.common.data.overpass_processing import get_polygons_from_closed_ways
+
+
+
+
+RIVERS_CACHE = GpxDataCacheHandler(name='rivers', extension='.pkl')
+
+
+def download_city_rivers(bounds: GpxBounds) -> Surface_Polygons:
```

Comment:
is the interior polygon part of the river actually used ?

Reply:
Yes, there are some islands on the seine river 😀

Reply:
This feature allows the code to draw the 2 branches on the river on multiple locations